yarn.lock

```diff
@@ -23,14 +23,14 @@ acorn-jsx@^3.0.0:
   dependencies:
     acorn "^3.0.4"
 
-acorn@4.0.4:
-  version "4.0.4"
-  resolved "https://registry.yarnpkg.com/acorn/-/acorn-4.0.4.tgz#17a8d6a7a6c4ef538b814ec9abac2779293bf30a"
-
 acorn@^3.0.4, acorn@^3.1.0:
   version "3.3.0"
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-3.3.0.tgz#45e37fb39e8da3f25baee3ff5369e2bb5f22017a"
 
+acorn@^5.0.1:
```

Comment:
This file will be axed, I imagine

Reply:
This has already been addressed in the "headless" branch I'm working on.  I'm still wrestling a bit with Firefox, but I'll clean that up and submit just for Chrome once this PR is merged.